tests/foreman/api/test_user.py

```diff
@@ -103,7 +99,7 @@ def setUpClass(cls):
         cls.anon_role = roles[0]
 
     @tier1
-    def test_create_with_role(self):
+    def test_positive_create(self):
```

Comment:
Oh wait, now i remember. Class name is `UserRoleTestCase` that's why i skipped `user` and `role` in test name. Anyway, `test_positive_create_with_role` is probably better as it avoids such confusion.